### PR TITLE
Fix the autotool Makefile, missing pht include

### PR DESCRIPTION
**Pull request by kaldoran**
I fixe the Makefile.am from src, cause we were missing the pht file in it.
Now we can create the makefile using autotools again.

( cmake was already update )

### CODE REVIEW

File: src/Makefile.am [hunk 1]
```diff
@@ -28,4 +29,5 @@ nobase_include_HEADERS = \
         ../include/opendht/securedht.h \
         ../include/opendht/dhtrunner.h \
         ../include/opendht/default_types.h \
-        ../include/opendht/rng.h
+        ../include/opendht/rng.h \
+		../include/opendht/indexation/pht.h
```
Comment (sim590) [inline src/Makefile.am]: Make sure that lines are aligned together :3

Comment (kaldoran) [inline src/Makefile.am]: I saw that, tabulation instead of space, i'll edit that

**Comment by kaldoran** [comment]
@sim590 Here i need you for merge if you are ok with it

**Comment by sim590** [comment]
Alright @kaldoran. Since this is a small change, you could squash all of your commits into a single one. Then, I'll merge it.

**Comment by kaldoran** [comment]
@sim590 that's done :)

**Pull request merged**